src/components/CountryFilter.tsx

```
@@ -1,35 +1,38 @@
-  name: string;
-  region: string;
-}
 
-export const CountryFilter = ({
-  filteredCountries,
-  onRegionChange,
-}: {
-  filteredCountries: Country[];
-  onRegionChange: (region: string) => void;
-}) => {
+export const CountryFilter = ({}: {}) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)